spec/lib/vmdb/loggers_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe Vmdb::Loggers do
+  let(:log_file) { Rails.root.join("log", "foo.log").to_s }
+  describe "#create_multicast_logger" do
+    it "defaults the lower level loggers to `DEBUG`" do
+      log = described_class.send(:create_multicast_logger, log_file)
+
+      expect(log.loggers.first.level).to eq(Logger::DEBUG)
+      expect(log.loggers.to_a.size).to   eq(1)
+    end
+
+    context "in a container environment" do
+      around(:each) { |example| in_container_env(example) }
```

Comment:
The :each is redundant as it's the default, so please remove (same below).

Reply:
Sure, no prob.  Again, though, ripped off from here:  https://github.com/ManageIQ/manageiq/commit/d098bc0

Where `around(:each)` was used.